gpflow/config/__config__.py

```diff
@@ -124,12 +139,17 @@ def set_default_jitter(value: float):
     set_config(replace(config(), jitter=value))
 
 
-def set_default_summary_fmt(value: str):
-    formats = tabulate.tabulate_formats + ['notebook', None]
-    if value not in formats:
-        raise ValueError(f"Summary does not support '{value}' format")
+def set_default_positive_bijector(value: str):
+    if not isinstance(value, str):
+        raise TypeError(f"Expected a string, but got a {type(value)}")
```

Comment:
This error message could be a bit confusing - it doesn't actually expect "just a string", but a specific set of values. As a user, it would be more helpful to directly get the message "It has to be 'A' or 'B', not 5.0". Could deal with the lowering by "if isinstance(value, str): value = value.lower()"; if it's not a string it still won't be in the set of valid options.

Reply:
(Similar to set_default_summary_fmt)